spec/features/dashboard_spec.rb

```diff
@@ -59,6 +59,26 @@
       expect(page).to have_content(lucky_user.user.email)
       expect(page).to_not have_content(unlucky_user.user.email)
     end
+
+    scenario 'feed displays 5 recent events' do
+      FactoryGirl.create_list(:status_pair, 10)
+      user = FactoryGirl.create(:user)
+      FactoryGirl.create_list(:status_zone, 2, user: user)
+      FactoryGirl.create_list(:status_help, 2, user: user)
+
+      sign_in_as(user)
+      visit user_profile_path(user)
+
+      fill_in "project_project", with: "Book reviews in space"
+      click_on 'Update project'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.